djangoevents/__init__.py

```diff
@@ -19,9 +21,20 @@
     'entity_mutator',
     'singledispatch',
     'publish',
+    'store_event'
     'subscribe',
     'unsubscribe',
     'subscribe_to',
     'BaseEntity',
     'EventSourcingWithDjango'
 ]
+
+
+def publish(event):
+    warnings.warn("`publish` is depreciated. Please switch to: `store_event`", DeprecationWarning)
+    return es_publish(event)
+
+
+def store_event(event):
```

Comment:
it is broadcasting event, we just don't have custom event subscriber. I don't see why we should discourage usage of subscribe. If we don't want use eventsourcing's subscribe, we should create a abstraction of publish and subscribe and we can swap the implementation of publish/subscribe.